en_US/extensions/plugins.md

```diff
@@ -65,12 +95,194 @@ my_emqx_plugin
 4 directories, 16 files
 ```
 
-Review the `rebar.config` file and adjust it as necessary for your plugin's requirements.
+- `src`: Contains the code for the plugin’s OTP application.
+
+- `priv`: Holds the plugin’s configuration files and schema (with example files).
+- `rebar.config`: The `rebar3` configuration file used to build the application and package it into a release.
+- `Makefile`: The entry point for building the plugin.
+- `scripts`: Helper scripts for the `Makefile`.
+- `README.md`: Documentation placeholder.
+- `LICENSE`: Sample license file for the plugin.
+- Review the `rebar.config` file and adjust it as necessary for your plugin's requirements.
 
 The project also includes an example module demonstrating how to add custom `emqx ctl` commands (`emqx_cli_demo.erl`).
```

Comment:
This is not actual (the file name is different), also we describe `_cli.erl` file further.